### PR TITLE
New alert messages UI

### PR DESCRIPTION
**Pull request by AlexandreSi**
image
image

image

### CODE REVIEW

**Review by 4ian — APPROVED** [review]

**Comment by arthuro555** [comment]
Wow very beautiful 👏

**Comment by 4ian** [comment]
Agree, this is just super clean and nicely done.

File: newIDE/app/src/AnnouncementsFeed/index.js [hunk 1]
```diff
@@ -56,13 +58,58 @@ export const AnnouncementsFeed = ({
...
       {({ i18n }) => (
         <Line noMargin={!addMargins}>
           <ColumnStackLayout noMargin={!addMargins} expand>
             {displayedAnnouncements.map(announcement => {
               const { buttonLabelByLocale, buttonUrl } = announcement;
+              const {
+                title,
+                message,
+                isMarkdownImageOnly,
+              } = getAnnouncementContent(i18n, announcement);
```
Comment (4ian) [inline newIDE/app/src/AnnouncementsFeed/index.js:112]: Not sure if worth it, but you could have useMemo the result of this by mapping before the render.
Maybe this is premature optimization though

**Review by ClementPasteau — APPROVED** [review]
lgtm

File: newIDE/app/src/AnnouncementsFeed/index.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/AnnouncementsFeed/index.js:112]: I think this optimization will be useful if we extract the rendered announcement into a new component

**Pull request merged**